modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/edit_public_render_parameters.jsp

```diff
@@ -23,6 +23,7 @@ List<PublicRenderParameterConfiguration> publicRenderParameterConfigurations = (
 Set<PublicRenderParameter> publicRenderParameters = (Set<PublicRenderParameter>)request.getAttribute(WebKeys.PUBLIC_RENDER_PARAMETERS);
 %>
 
+<h1>edit_public_render_parameters</h1>
```

Comment:
Is this supposed to be here?

Reply:
<img src="https://media4.giphy.com/media/l0IydbsUbimmcv2yQ/giphy.gif"/>